packages/orbit-components/src/CountryFlag/CountryFlag.stories.tsx

```diff
@@ -32,10 +31,9 @@ const meta: Meta<typeof CountryFlag> = {
         type: "select",
       },
     },
-    role: {
-      options: ["img", "presentation", "none"],
+    name: {
```

Comment:
Why did you remove `role` field from SB? I understand there's no visual change for the users - just DOM change, but we included other "no-visible" fields in other stories.

Reply:
To incentivize using name more than role because empty name is more recommendable than using role